android/app/src/main/AndroidManifest.xml

```diff
@@ -10,6 +12,12 @@
       android:roundIcon="@mipmap/ic_launcher_round"
       android:allowBackup="false"
       android:theme="@style/AppTheme">
+      <meta-data
+       android:name="com.google.android.geo.API_KEY"
+       android:value="AIzaSyDmoUbPJahCHn1yZSMezO9Z-dPhUOBq9Uw"/>
```

Comment:
please, hide API keys from AndroidManifest.xml using local.properties, also check the documentation about best practices while configuring an API Key on Google Maps: https://developers.google.com/maps/api-key-best-practices

Reply:
@fernandovfilho  you can change `value` to `@strings/google_maps_api_key`

Reply:
@lucasmontano In that case, isn't better to keep all the keys, even if they are used by native Android, inside the `.env` file in the root folder? So it is possible to keep only one file with everything centralized.
What do you think @fernandovfilho, @jarthursantos and @diego3g?

Reply:
It's a good idea

Reply:
It's a good idea, we can do is create a script, when before to build app create a file and throw to inside the directory "android" as local.proprities with the configuratons of file env ".env".
What you think about that? @csorlandi